src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+    const { asFragment } = render(<Search />);
+
+    it("renders correctly", () => {
+        expect(asFragment()).toMatchSnapshot();
+    });
+});
```

Comment:
You should add more tests to verify your app works as expected, here are the tests I would add for this component:
- As a basic test I would at least render `Search` and check the elements are there as expected. You can use `screen.getByText` to check the button element. To check the input element you would have to make changes to your input, if you add a placeholder value (e.g. `placeholder="search"`) you could use `screen.getByPlaceholderText`, otherwise you'd need to add a data test id (e.g. `data-testid="search-input"`) and then use `screen.getByTestId`. Note that I've used `screen.`, which can be imported from `"@testing-library/react"`
- If your up for the challenge, you can also test the search functionality - this would be quite a bit harder and would involve using mocks and spys, as well as `fireEvent` and `waitFor` from the `"@testing-library/react"`

note: you should pass the prop `setSearchResults` when you render `Search`, i.e.
```
const mockSetSearchResults = jest.fn()
render(<Search setSearchResults={mockSetSearchResults} />)
```